math/straightlines/straightlines.go

```diff
@@ -0,0 +1,54 @@
+package straightlines
```

Comment:
I think this whole algorithm and test can be moved to the geometry package inside the math directory. Otherwise all good.

Reply:
It seems that the PR with `geometry` package didn't get merged. Could you then just change the package and folder name to geometry instead of `straightlines`

Reply:
I think no, only folder and package. There will be many algorithms in one package

Reply:
@tjgurwara99 @siriak The suggested changes have been made.